spec/order_spec.rb

```diff
@@ -0,0 +1,42 @@
+require './docs/order'
+describe Order do
+  let(:dishes_class) { double(:dishes_class) }
+  let(:dishes) { double(:dishes) }
+  describe '#select' do
+    # xit 'responsible for creating new dishes' do
+    #   allow(dishes_class).to receive(:new).and_return(dishes)
+    #   allow(dishes).to receive(:price)
+    #   order = Order.new(dishes_class)
+    #   order.select(dish)
+    # end
+    it 'can respond to select with an argument' do
+      expect(subject).to respond_to(:select).with(1).argument
+    end
+    it 'can add prices to total' do
+      allow(dishes_class).to receive(:new).and_return(dishes)
+      allow(dishes).to receive(:price).and_return(10)
```

Comment:
Nice use of dependency injection 💯 

Small point, and I can see why you are doing it this way because it's a similar approach to the `Till` code we were looking at last week, but we probably can just give an `instance` of `Dishes` to `Order` rather than a reference of the class. The difference is whether we want to create a new instance of `Dishes` everytime we make an order, or whether our instance of `Order` can refer to the same `Dishes` instance